Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+MAINTAINER Shikher Verma "root@shikherverma.com"
+RUN apt-get update -y --fix-missing
+RUN apt-get install -y python python-pip python-dev build-essential git nodejs npm ruby ruby-dev
```

Comment:
You seem to use nodejs to build/install gem stuff. We probably could use a mullti-stage build in this case.

Reply:
`gem` comes with `ruby` and `ruby-dev` was needed to install `sass` gem. Maybe I could ditch this in favour of some `apt` package.  
`nodejs` is for `[npm dependencies](https://github.com/in-toto/layout-web-tool/blob/wizard-beta/gulpfile.js)` not sure if they would be available on `apt` but I'll look.